indico/htdocs/js/indico/widgets/person_link_widget.js

```diff
@@ -48,12 +49,83 @@
         var $noOtherPlaceholder = $fieldDisplay.find('#no-other-placeholder-' + options.fieldId);
         var $buttonAddExisting = $('#add-existing-' + options.fieldId);
         var $buttonAddNew = $('#add-new-' + options.fieldId);
+        var $buttonAlphaOrder = $fieldDisplay.find('.alpha-order-switch');
+
+        function updatePersonOrder() {
+            var people = _.map($fieldDisplay.find('.person-row'), function(e) { return $(e).data('person'); });
+            people.forEach(function(person, i) {
+                person.displayOrder = i + 1;
+                $field.principalfield('set', person.id, {displayOrder: person.displayOrder});
+            });
+        }
+
+        function setupList(entryType, $list, sortedPeople) {
+            var isSortable = options.sort === true;
```

Comment:
Any good reason not to? At least I usually use `=== true` only if I care that it's true and not something else that's truthy (more often I do that for `false` where I want to avoid handling `undefined` the same way).